tests/gtest/avifsampletransformtest.cc

```diff
@@ -28,7 +28,7 @@ class AvifExpression : public avifSampleTransformExpression {
     token.type = AVIF_SAMPLE_TRANSFORM_INPUT_IMAGE_ITEM_INDEX;
     token.inputImageItemIndex = inputImageItemIndex;
   }
-  void AddOperator(uint8_t op) {
+  void AddOperator(avifSampleTransformTokenType op) {
```

Comment:
(This is an unrelated issue.) I noticed that the local variable `token` declared at line 60 shadows the data member `token` of the base class `avifSampleTransformExpression`. It would be good to rename the local variable.

Reply:
The data member is named `tokens` in `avifSampleTransformExpression`:

https://github.com/AOMediaCodec/libavif/blob/40b5589b003ca11a63572a470ac8744404fec2f9/include/avif/internal.h#L204

So there is no shadowing.

Reply:
You are right. I made a mistake when using Code Search, but I don't know what I did wrong :-). Sorry!